microsoft-edge/devtools-guide-chromium/performance/performance-tool-overview.md

```diff
@@ -0,0 +1,410 @@
+
+* CPU throttling: Simulate slower CPU speeds.  See [Throttle the CPU while recording](../performance/reference.md#throttle-the-cpu-while-recording) in _Performance features reference_.  
+
+* Network throttling: Simulate slower network speeds.  See [Throttle the network while recording](../performance/reference.md#throttle-the-network-while-recording) in _Performance features reference_.
+
+
+<!-- ------------------------------ -->
+#### Analyze a performance report
+<!-- https://developer.chrome.com/docs/devtools/performance/overview#analyze_a_performance_report -->
+
+For a complete guide on how to use the **Performance** tool, see [Analyze a performance recording](../performance/reference.md#analyze-a-performance-recording) in _Performance features reference_.
+
+
+To navigate a performance report:
+
+* See [Navigate the recording](../performance/reference.md#navigate-the-recording) in _Performance features reference_.
+
+* See [Search activities](../performance/reference.md#search-activities) in _Performance features reference_.
+
+* See [Track event initiators](../performance/reference.md#track-event-initiators) in _Performance features reference_.
+
+
+To focus on what matters for your workflow:
+
+* [Change the order of tracks and hide them](../performance/reference.md#change-the-order-of-tracks-and-hide-them)
+
+* [Hide functions and their children in the flame chart](../performance/reference.md#hide-functions-and-their-children-in-the-flame-chart)
+
+* [Create breadcrumbs and jump between zoom levels](./reference.md#create-breadcrumbs-and-jump-between-zoom-levels) in _Performance features reference_.
+
+
+To learn about the **Bottom-up**, **Call tree**, and **Event log** tabs:
+
+* [View activities in a table](./reference.md#view-activities-in-a-table) in _Performance features reference_.
+
+
+To analyze a performance report:
+
+* [View main thread activity](./reference.md#view-main-thread-activity) in _Performance features reference_.
+* [Read the flame chart](./reference.md#read-the-flame-chart) in _Performance features reference_.
+* [View a screenshot](./reference.md#view-a-screenshot) in _Performance features reference_.
+* [View memory metrics](./reference.md#view-memory-metrics) in _Performance features reference_.
+* [View the duration of a portion of a recording](./reference.md#view-the-duration-of-a-portion-of-a-recording) in _Performance features reference_.
+* [Analyze CSS selector performance during Recalculate Style events](./selector-stats.md).
+* [Analyze frames per second (FPS)](./reference.md#analyze-frames-per-second-fps) in _Performance features reference_.
+* [Performance recording event reference](./performance-reference.md)
+<!-- todo: link to Edge docset
```

Comment:
reveal this link pointing to Chrome docs?  plan to copy article to Edge docs & link to there?  

**Profile Node.js performance with the Performance tool**
https://developer.chrome.com/docs/devtools/performance/nodejs

Reply:
removed hidden comment